ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -76,9 +80,9 @@ def _check_param_keys(self):
         # Check that batch size is greater than sequence length. Else, throw
         # an exception.
         if (
-            self.trainer_parameters["sequence_length"]
-            > self.trainer_parameters["batch_size"]
-            and self.trainer_parameters["use_recurrent"]
+            self.trainer_settings["sequence_length"]
+            > self.trainer_settings["batch_size"]
+            and self.trainer_settings["use_recurrent"]
```

Comment:
I think this can be:
```suggestion
            self.trainer_settings.sequence_length
            > self.trainer_settings.batch_size
            and self.trainer_settings.use_recurrent
```

Reply:
Ah, this method isn't actually used anymore, the check is done in `settings.py` in `_check_batch_size_seq_length`. I've removed them from PPO and SAC.